vendor/assets/javascripts/fixedsticky.js

```diff
@@ -0,0 +1,191 @@
+;(function( win, $ ) {
+
+  function featureTest( property, value, noPrefixes ) {
+    // Thanks Modernizr! https://github.com/phistuck/Modernizr/commit/3fb7217f5f8274e2f11fe6cfeda7cfaf9948a1f5
+    var prop = property + ':',
+      el = document.createElement( 'test' ),
+      mStyle = el.style;
+
+    if( !noPrefixes ) {
+      mStyle.cssText = prop + [ '-webkit-', '-moz-', '-ms-', '-o-', '' ].join( value + ';' + prop ) + value + ';';
```

Comment:
Should the `';'` in the join method call be a `':'` instead?  It looks like you're setting a CSS property and value with this statement, right?